src/typescript/azure-openapi-validator/tests/individual-azure-tests.ts

```diff
@@ -28,10 +28,11 @@ import { LicenseHeaderMustNotBeSpecified } from '../rules/LicenseHeaderMustNotBe
     const messages: Message[] = await collectTestMessagesFromValidator(fileName, OpenApiTypes.arm, MergeStates.individual);
```

Comment:
nit: No useful changes

Reply:
Moved an existing test to its new designation as a documentation violation (the "no description" test). If you would rather keep the original designation, then I'll move it back here.